src/Disks/registerDisks.h

```diff
@@ -1,11 +1,37 @@
 #pragma once
 
+#include <Core/Types.h>
+
 namespace DB
 {
 
-/// @param global_skip_access_check - skip access check regardless regardless
-///                                   .skip_access_check config directive (used
-///                                   for clickhouse-disks)
-void registerDisks(bool global_skip_access_check);
+enum class DiskFlag : size_t
+{
```

Comment:
```suggestion
enum class DiskFlag
```
No need to specify explicit type, enum is internal to CH

Reply:
I also think we could rename it to DiskStartupFlag to resolve ambiguity